### ISSUE TITLE
Layer attribute order

### ISSUE THREAD

**Issue by karelMT** [gislab-npo/gisquick#74]
QGIS support custom attribute order in layer, but GQ always show default attribute order. Is it possible improve it in new version?

**Comment by marcel-dancak** [comment]
I'm not sure what kind of order do you mean. Is it order of columns in attribute table, which can be in QGIS configured as described here https://gis.stackexchange.com/questions/167493/rearranging-display-order-of-attribute-table-fields-using-qgis/241315#241315, or do you mean sorting of rows (results) by selected attribute?

**Comment by karelMT** [comment]
Yes, I mean this. I used function "Drag and drop designer" to set my own attribution order (without group), but it didn't do anything in GQ and order is default. I also tried "Select organize columns" in attribute table without any result too.

Is possible to change it?
